pkg/k8s/api.go

```diff
@@ -185,6 +185,22 @@ func (kubeAPI *KubernetesAPI) GetAllNamespacesWithExtensionLabel(ctx context.Con
 	return namespaces.Items, nil
 }
 
+// CheckIfExtensionExists checks if an extension is installed in the cluster
```

Comment:
Instead of adding a new function how about using `GetNamespaceWithExtensionLabel`?

Its final return could be changed to return a `NotFound` error and we could use that to check the type of `err` if `ns` is `nil` like we do in other places.

I'm thinking the return would be:
```
return nil, errors.NewNotFound(corev1.Resource("namespace"), value)
```

Reply:
Makes sense. Updated!